gbpservice/neutron/nsf/core/cfg.py

```diff
@@ -0,0 +1,53 @@
+from oslo_config import cfg
```

Comment:
This file is not part of config_agent ,removed core and proxy from config_agent_modules branch